playbook.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: all
```

Comment:
It's not necessary.
See [this](https://travis-ci.org/kostyrevaa/wildfly/builds/180369421) successful Travis build.